tweets/2020/06/gilliano-talk-at-python-user-group-pernambuco.tweet

```diff
@@ -0,0 +1 @@
+Our engineer Gilliano Menezes (@gillianomenezes) will give a talk [in pt-br 🇧🇷] about S.O.L.I.D. principles at the 66º meeting of Python User Group Pernambuco next Saturday at 23:00 UTC!🚀 The event will be online in YT: https://youtu.be/BHSsjnN1jV0
```

Comment:
Great!

Some minor suggestions:
Instead of 23:00 say 11pm since it's more common here, and change `in YT` to `on YouTube` (to also make it explicit)

🎉

Reply:
I'll do it. Thanks!